tests/transform-header-lowercase/test.yaml

```diff
@@ -0,0 +1,40 @@
+requires:
+  min-version: 7.0.0
```

Comment:
needs to either target 8, or check for the new file

Reply:
Indeed